chore: Bump github.com/containers/podman from v4 to v5

rpoisel opened the pull request:
I just completed the [Quick Start](https://github.com/containers/fetchit/blob/main/docs/quick_start.rst) example successfully with the locally built image based on this version.

Crosswind commented:
I have rebased the changes from @rpoisel, updated to podman v5.5.0, re-used the version variable for the ref and also changed the job names in the Workflow file from v4 to v5 to be consistent. I can open my own PR or otherwise, here's the patch:

```patch
diff --git a/.github/workflows/docker-image.yml b/.github/workflows/docker-image.yml
index 2487336..329a366 100644
--- a/.github/workflows/docker-image.yml
+++ b/.github/workflows/docker-image.yml
@@ -8,11 +8,11 @@ on:

 env:
   CGO_ENABLED: 0
-  PODMAN_VER: v5.4.2
+  PODMAN_VER: v5.5.0

 jobs:

-  build-podman-v4:
+  build-podman-v5:
     env:
       CGO_ENABLED: 1 # CGO is required for podman
     runs-on: ubuntu-latest
@@ -30,7 +30,7 @@ jobs:
         if: steps.cache-podman-bin.outputs.cache-hit != 'true'
         with:
           repository: containers/podman
-          ref: v5.4.2
+          ref: ${{ env.PODMAN_VER }}

       - uses: actions/setup-go@v4
         if: steps.cache-podman-bin.outputs.cache-hit != 'true'
@@ -54,7 +54,7 @@ jobs:
         if: steps.cache-podman-bin.outputs.cache-hit != 'true'
         run: sudo apt install -y libsystemd-dev libseccomp-dev pkg-config golang-github-proglottis-gpgme-dev

-      - name: Build podman v4
+      - name: Build podman v5
         if: steps.cache-podman-bin.outputs.cache-hit != 'true'
         run: make binaries
```

rpoisel commented:
> I have rebased the changes from @rpoisel, updated to podman v5.5.0, re-used the version variable for the ref and also changed the job names in the Workflow file from v4 to v5 to be consistent. I can open my own PR or otherwise, here's the patch:
> ...
> Also, the GitHub Actions ran fine on my repository after all the changes have been applied.

Thanks for keeping an eye on this and for trying to update the Podman build dependency to the latest version.

@Crosswind, have you tried to update the actual go dependencies as well?

```shell
go get github.com/containers/podman/v5@v5.5.0
go mod tidy
go mod vendor
```

After performing the upgrade, I was unable to build fetchit:

```shell
podman build . --file Dockerfile --tag quay.io/fetchit/fetchit-amd:latest
...
go build -mod=vendor -tags 'include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo exclude_graphdriver_btrfs' -ldflags "-X k8s.io/component-base/version.gitMajor=0 -X k8s.io/component-base/version.gitMajor=0 -X k8s.io/component-base/version.gitMinor=0 -X k8s.io/component-base/version.gitVersion=v0.0.0 -X k8s.io/component-base/version.gitTreeState=clean -X k8s.io/client-go/pkg/version.gitMajor=0 -X k8s.io/client-go/pkg/version.gitMinor=0 -X k8s.io/client-go/pkg/version.gitVersion=v0.0.0 -X k8s.io/client-go/pkg/version.gitTreeState=clean " -o '_output/bin/linux_amd64/fetchit' github.com/containers/fetchit/cmd/fetchit
# github.com/sigstore/sigstore-go/pkg/root
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:33:26: sc.signingConfig.GetCaUrl undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method GetCaUrl)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:37:26: sc.signingConfig.GetOidcUrl undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method GetOidcUrl)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:41:26: sc.signingConfig.GetTlogUrls undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method GetTlogUrls)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:45:9: cannot use sc.signingConfig.GetTsaUrls() (value of type []*"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".Service) as []string value in return statement
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:49:19: sc.signingConfig.CaUrl undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method CaUrl)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:54:19: sc.signingConfig.OidcUrl undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method OidcUrl)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:59:19: sc.signingConfig.TlogUrls undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method TlogUrls)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:64:54: sc.signingConfig.TlogUrls undefined (type *"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".SigningConfig has no field or method TlogUrls)
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:69:29: cannot use tsaURLs (variable of type []string) as []*"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".Service value in assignment
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:74:62: cannot use tsaURLs (variable of type []string) as []*"github.com/sigstore/protobuf-specs/gen/pb-go/trustroot/v1".Service value in argument to append
vendor/github.com/sigstore/sigstore-go/pkg/root/signing_config.go:74:62: too many errors
```

Upgrading all sigstore dependencies to the latest version fixes the compilation errors:

```shell
go get github.com/sigstore/...
go mod tidy
go mod vendor
make _build_local
```

However, this only works with go version 1.24. The `registry.access.redhat.com/ubi8/go-toolset`  images only contain go version 1.23 at the moment.

Therefore, I just edited `.github/workflows/docker-image.yml` to let the `PODMAN_VER` variable point to `v5.4.2`. That's the version which is compatible to go version 1.23 with all its dependencies.

Let's focus on upgrading to v5.5.0 in a later PR as soon as the build images support go version 1.24, I'd suggest.

Crosswind commented:
@rpoisel here’s my build when upgrading to v5.5.0: https://github.com/Crosswind/fetchit/actions/runs/15344060475/job/43176306430
The overall build is failing because it was trying to run steps on main that can only run on this repo. 
The build step (that still has v4 in its name) looks successful though. 

In any case, I agree that we don’t have to crunch all the changes in here. If sticking with 5.4.2 gets us moving quicker, let’s go for it. 
WRT the ubi8 image, I was also thinking about moving that to its ubi9 version.

cooktheryan commented:
@rpoisel could you update the fedora image version. After that i think we are good

```
➜  ansible git:(Embedded-Focus-dev-upgrade-podman-from-v4-to-v5) ✗ podman build -f Dockerfile .
STEP 1/5: FROM quay.io/fedora/fedora:40
Trying to pull quay.io/fedora/fedora:40...
Getting image source signatures
Copying blob sha256:643282ea968d5b2b35cc0ea59fd907b0b409578917e6e11c6067db575f65ccf5
Copying config sha256:1b01d806e510ec5280eb2bf53d4e67ae58939dfb240d6af35bc39f8b2e42a260
Writing manifest to image destination
STEP 2/5: ARG ARCH=$ARCH
--> cc7e8440eba7
STEP 3/5: ARG MAKE_TARGET=cross-build-linux-$ARCH-ansible
--> f1ed8fde9060
STEP 4/5: RUN yum -y install openssh-clients ansible && yum clean all
Fedora 40 - aarch64                             7.0 MB/s |  19 MB     00:02
Fedora 40 openh264 (From Cisco) - aarch64       3.0 kB/s | 1.4 kB     00:00
Fedora 40 - aarch64 - Updates                   7.1 MB/s |  12 MB     00:01
Dependencies resolved.
================================================================================
 Package                 Arch       Version                    Repository  Size
================================================================================
Installing:
 ansible                 noarch     9.13.0-1.fc40              updates     54 M
 openssh-clients         aarch64    9.6p1-2.fc40               updates    744 k
Installing dependencies:
 ansible-core            noarch     2.16.14-1.fc40             updates    3.7 M
 fmt                     aarch64    10.2.1-5.fc40              updates    121 k
 libcbor                 aarch64    0.11.0-1.fc40              fedora      33 k
 libdnf5                 aarch64    5.1.17-4.fc40              updates    908 k
 libedit                 aarch64    3.1-54.20250104cvs.fc40    updates    107 k
 libfdisk                aarch64    2.40.2-1.fc40              updates    157 k
 libfido2                aarch64    1.14.0-4.fc40              fedora      96 k
 libutempter             aarch64    1.2.1-13.fc40              fedora      27 k
 openssh                 aarch64    9.6p1-2.fc40               updates    423 k
 python3-argcomplete     noarch     3.6.2-2.fc40               updates     97 k
 python3-cffi            aarch64    1.16.0-4.fc40              fedora     313 k
 python3-cryptography    aarch64    41.0.7-1.fc40              fedora     1.2 M
 python3-jinja2          noarch     3.1.6-1.fc40               updates    487 k
 python3-markupsafe      aarch64    2.1.3-4.fc40               fedora      30 k
 python3-packaging       noarch     23.2-4.fc40                fedora     125 k
 python3-ply             noarch     3.11-23.fc40               fedora     136 k
 python3-pycparser       noarch     2.20-14.fc40               fedora     157 k
 python3-pyyaml          aarch64    6.0.1-14.fc40              fedora     225 k
 python3-resolvelib      noarch     1.0.1-4.fc40               fedora      45 k
 util-linux              aarch64    2.40.2-1.fc40              updates    1.2 M
Installing weak dependencies:
 python3-libdnf5         aarch64    5.1.17-4.fc40              updates    1.4 M

Transaction Summary
================================================================================
Install  23 Packages

Total download size: 65 M
Installed size: 396 M
Downloading Packages:
(1/23): libcbor-0.11.0-1.fc40.aarch64.rpm       132 kB/s |  33 kB     00:00
(2/23): libutempter-1.2.1-13.fc40.aarch64.rpm    98 kB/s |  27 kB     00:00
(3/23): python3-cffi-1.16.0-4.fc40.aarch64.rpm  3.6 MB/s | 313 kB     00:00
(4/23): python3-cryptography-41.0.7-1.fc40.aarc 6.8 MB/s | 1.2 MB     00:00
(5/23): python3-markupsafe-2.1.3-4.fc40.aarch64 262 kB/s |  30 kB     00:00
(6/23): python3-packaging-23.2-4.fc40.noarch.rp 3.6 MB/s | 125 kB     00:00
(7/23): python3-ply-3.11-23.fc40.noarch.rpm     3.0 MB/s | 136 kB     00:00
(8/23): libfido2-1.14.0-4.fc40.aarch64.rpm      193 kB/s |  96 kB     00:00
(9/23): python3-pycparser-2.20-14.fc40.noarch.r 4.3 MB/s | 157 kB     00:00
(10/23): python3-pyyaml-6.0.1-14.fc40.aarch64.r 5.1 MB/s | 225 kB     00:00
(11/23): python3-resolvelib-1.0.1-4.fc40.noarch 997 kB/s |  45 kB     00:00
(12/23): fmt-10.2.1-5.fc40.aarch64.rpm          589 kB/s | 121 kB     00:00
(13/23): libdnf5-5.1.17-4.fc40.aarch64.rpm      1.5 MB/s | 908 kB     00:00
(14/23): libedit-3.1-54.20250104cvs.fc40.aarch6 983 kB/s | 107 kB     00:00
(15/23): ansible-core-2.16.14-1.fc40.noarch.rpm 3.9 MB/s | 3.7 MB     00:00
(16/23): libfdisk-2.40.2-1.fc40.aarch64.rpm     1.6 MB/s | 157 kB     00:00
(17/23): openssh-9.6p1-2.fc40.aarch64.rpm       3.5 MB/s | 423 kB     00:00
(18/23): python3-argcomplete-3.6.2-2.fc40.noarc 1.1 MB/s |  97 kB     00:00
(19/23): python3-jinja2-3.1.6-1.fc40.noarch.rpm 2.9 MB/s | 487 kB     00:00
(20/23): openssh-clients-9.6p1-2.fc40.aarch64.r 1.4 MB/s | 744 kB     00:00
(21/23): python3-libdnf5-5.1.17-4.fc40.aarch64. 3.7 MB/s | 1.4 MB     00:00
(22/23): util-linux-2.40.2-1.fc40.aarch64.rpm   1.5 MB/s | 1.2 MB     00:00
(23/23): ansible-9.13.0-1.fc40.noarch.rpm       9.1 MB/s |  54 MB     00:05
--------------------------------------------------------------------------------
Total                                           9.5 MB/s |  65 MB     00:06
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1
  Installing       : fmt-10.2.1-5.fc40.aarch64                             1/23
  Installing       : libdnf5-5.1.17-4.fc40.aarch64                         2/23
  Installing       : python3-libdnf5-5.1.17-4.fc40.aarch64                 3/23
  Installing       : python3-argcomplete-3.6.2-2.fc40.noarch               4/23
  Installing       : libfdisk-2.40.2-1.fc40.aarch64                        5/23
  Installing       : libedit-3.1-54.20250104cvs.fc40.aarch64               6/23
  Installing       : python3-resolvelib-1.0.1-4.fc40.noarch                7/23
  Installing       : python3-pyyaml-6.0.1-14.fc40.aarch64                  8/23
  Installing       : python3-ply-3.11-23.fc40.noarch                       9/23
  Installing       : python3-pycparser-2.20-14.fc40.noarch                10/23
  Installing       : python3-cffi-1.16.0-4.fc40.aarch64                   11/23
  Installing       : python3-cryptography-41.0.7-1.fc40.aarch64           12/23
  Installing       : python3-packaging-23.2-4.fc40.noarch                 13/23
  Installing       : python3-markupsafe-2.1.3-4.fc40.aarch64              14/23
  Installing       : python3-jinja2-3.1.6-1.fc40.noarch                   15/23
  Installing       : ansible-core-2.16.14-1.fc40.noarch                   16/23
  Running scriptlet: libutempter-1.2.1-13.fc40.aarch64                    17/23
  Installing       : libutempter-1.2.1-13.fc40.aarch64                    17/23
  Installing       : util-linux-2.40.2-1.fc40.aarch64                     18/23
  Running scriptlet: util-linux-2.40.2-1.fc40.aarch64                     18/23
  Installing       : openssh-9.6p1-2.fc40.aarch64                         19/23
  Installing       : libcbor-0.11.0-1.fc40.aarch64                        20/23
  Installing       : libfido2-1.14.0-4.fc40.aarch64                       21/23
  Installing       : openssh-clients-9.6p1-2.fc40.aarch64                 22/23
  Running scriptlet: openssh-clients-9.6p1-2.fc40.aarch64                 22/23
  Installing       : ansible-9.13.0-1.fc40.noarch                         23/23
  Running scriptlet: ansible-9.13.0-1.fc40.noarch                         23/23

Installed:
  ansible-9.13.0-1.fc40.noarch
  ansible-core-2.16.14-1.fc40.noarch
  fmt-10.2.1-5.fc40.aarch64
  libcbor-0.11.0-1.fc40.aarch64
  libdnf5-5.1.17-4.fc40.aarch64
  libedit-3.1-54.20250104cvs.fc40.aarch64
  libfdisk-2.40.2-1.fc40.aarch64
  libfido2-1.14.0-4.fc40.aarch64
  libutempter-1.2.1-13.fc40.aarch64
  openssh-9.6p1-2.fc40.aarch64
  openssh-clients-9.6p1-2.fc40.aarch64
  python3-argcomplete-3.6.2-2.fc40.noarch
  python3-cffi-1.16.0-4.fc40.aarch64
  python3-cryptography-41.0.7-1.fc40.aarch64
  python3-jinja2-3.1.6-1.fc40.noarch
  python3-libdnf5-5.1.17-4.fc40.aarch64
  python3-markupsafe-2.1.3-4.fc40.aarch64
  python3-packaging-23.2-4.fc40.noarch
  python3-ply-3.11-23.fc40.noarch
  python3-pycparser-2.20-14.fc40.noarch
  python3-pyyaml-6.0.1-14.fc40.aarch64
  python3-resolvelib-1.0.1-4.fc40.noarch
  util-linux-2.40.2-1.fc40.aarch64

Complete!
17 files removed
--> f68706ef2762
STEP 5/5: ADD ansible.cfg /etc/ansible/
COMMIT
--> 12d2c1d38348
12d2c1d38348871c99ba6aad67c5758e6ed0158500634ed2e8acd57cae07061b



➜  ansible git:(Embedded-Focus-dev-upgrade-podman-from-v4-to-v5) ✗ podman build -f Dockerfile . --arch x86_64
STEP 1/5: FROM quay.io/fedora/fedora:40
Trying to pull quay.io/fedora/fedora:40...
Getting image source signatures
Copying blob sha256:ddb6b59dee108a2e9cd710e34faad56addf5612605f04c106c437d85755e08be
Copying config sha256:a7720e9fbcba245cd218ad9a7a1406b6164f5d30b385759b912130d4be830a45
Writing manifest to image destination
STEP 2/5: ARG ARCH=$ARCH
--> 432167c46eb2
STEP 3/5: ARG MAKE_TARGET=cross-build-linux-$ARCH-ansible
--> d67b8d75cf59
STEP 4/5: RUN yum -y install openssh-clients ansible && yum clean all
Fedora 40 - x86_64                              6.8 MB/s |  20 MB     00:02
Fedora 40 openh264 (From Cisco) - x86_64        3.1 kB/s | 1.4 kB     00:00
Fedora 40 - x86_64 - Updates                    7.8 MB/s |  13 MB     00:01
Last metadata expiration check: 0:00:01 ago on Thu Jun 12 19:03:26 2025.
Dependencies resolved.
================================================================================
 Package                 Arch      Version                     Repository  Size
================================================================================
Installing:
 ansible                 noarch    9.13.0-1.fc40               updates     54 M
 openssh-clients         x86_64    9.6p1-2.fc40                updates    744 k
Installing dependencies:
 ansible-core            noarch    2.16.14-1.fc40              updates    3.7 M
 fmt                     x86_64    10.2.1-5.fc40               updates    125 k
 libcbor                 x86_64    0.11.0-1.fc40               fedora      33 k
 libdnf5                 x86_64    5.1.17-4.fc40               updates    996 k
 libedit                 x86_64    3.1-54.20250104cvs.fc40     updates    105 k
 libfdisk                x86_64    2.40.2-1.fc40               updates    159 k
 libfido2                x86_64    1.14.0-4.fc40               fedora      98 k
 libutempter             x86_64    1.2.1-13.fc40               fedora      26 k
 openssh                 x86_64    9.6p1-2.fc40                updates    423 k
 python3-argcomplete     noarch    3.6.2-2.fc40                updates     97 k
 python3-cffi            x86_64    1.16.0-4.fc40               fedora     306 k
 python3-cryptography    x86_64    41.0.7-1.fc40               fedora     1.2 M
 python3-jinja2          noarch    3.1.6-1.fc40                updates    487 k
 python3-markupsafe      x86_64    2.1.3-4.fc40                fedora      30 k
 python3-packaging       noarch    23.2-4.fc40                 fedora     125 k
 python3-ply             noarch    3.11-23.fc40                fedora     136 k
 python3-pycparser       noarch    2.20-14.fc40                fedora     157 k
 python3-pyyaml          x86_64    6.0.1-14.fc40               fedora     233 k
 python3-resolvelib      noarch    1.0.1-4.fc40                fedora      45 k
 util-linux              x86_64    2.40.2-1.fc40               updates    1.2 M
Installing weak dependencies:
 python3-libdnf5         x86_64    5.1.17-4.fc40               updates    1.5 M

Transaction Summary
================================================================================
Install  23 Packages

Total download size: 65 M
Installed size: 379 M
Downloading Packages:
(1/23): libcbor-0.11.0-1.fc40.x86_64.rpm        357 kB/s |  33 kB     00:00
(2/23): libfido2-1.14.0-4.fc40.x86_64.rpm       711 kB/s |  98 kB     00:00
(3/23): libutempter-1.2.1-13.fc40.x86_64.rpm    177 kB/s |  26 kB     00:00
(4/23): python3-cffi-1.16.0-4.fc40.x86_64.rpm   3.7 MB/s | 306 kB     00:00
(5/23): python3-markupsafe-2.1.3-4.fc40.x86_64. 255 kB/s |  30 kB     00:00
(6/23): python3-packaging-23.2-4.fc40.noarch.rp 1.2 MB/s | 125 kB     00:00
(7/23): python3-cryptography-41.0.7-1.fc40.x86_ 7.4 MB/s | 1.2 MB     00:00
(8/23): python3-ply-3.11-23.fc40.noarch.rpm     1.2 MB/s | 136 kB     00:00
(9/23): python3-pycparser-2.20-14.fc40.noarch.r 1.3 MB/s | 157 kB     00:00
(10/23): python3-pyyaml-6.0.1-14.fc40.x86_64.rp 2.0 MB/s | 233 kB     00:00
(11/23): python3-resolvelib-1.0.1-4.fc40.noarch 496 kB/s |  45 kB     00:00
(12/23): fmt-10.2.1-5.fc40.x86_64.rpm           344 kB/s | 125 kB     00:00
(13/23): libdnf5-5.1.17-4.fc40.x86_64.rpm       2.2 MB/s | 996 kB     00:00
(14/23): libedit-3.1-54.20250104cvs.fc40.x86_64 381 kB/s | 105 kB     00:00
(15/23): libfdisk-2.40.2-1.fc40.x86_64.rpm      496 kB/s | 159 kB     00:00
(16/23): ansible-core-2.16.14-1.fc40.noarch.rpm 2.2 MB/s | 3.7 MB     00:01
(17/23): openssh-9.6p1-2.fc40.x86_64.rpm        1.5 MB/s | 423 kB     00:00
(18/23): python3-argcomplete-3.6.2-2.fc40.noarc 255 kB/s |  97 kB     00:00
(19/23): openssh-clients-9.6p1-2.fc40.x86_64.rp 1.5 MB/s | 744 kB     00:00
(20/23): python3-jinja2-3.1.6-1.fc40.noarch.rpm 1.2 MB/s | 487 kB     00:00
(21/23): python3-libdnf5-5.1.17-4.fc40.x86_64.r 2.1 MB/s | 1.5 MB     00:00
(22/23): util-linux-2.40.2-1.fc40.x86_64.rpm    2.0 MB/s | 1.2 MB     00:00
(23/23): ansible-9.13.0-1.fc40.noarch.rpm       8.9 MB/s |  54 MB     00:06
--------------------------------------------------------------------------------
Total                                           9.6 MB/s |  65 MB     00:06
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1
  Installing       : fmt-10.2.1-5.fc40.x86_64                              1/23
  Installing       : libdnf5-5.1.17-4.fc40.x86_64                          2/23
  Installing       : python3-libdnf5-5.1.17-4.fc40.x86_64                  3/23
  Installing       : python3-argcomplete-3.6.2-2.fc40.noarch               4/23
  Installing       : libfdisk-2.40.2-1.fc40.x86_64                         5/23
  Installing       : libedit-3.1-54.20250104cvs.fc40.x86_64                6/23
  Installing       : python3-resolvelib-1.0.1-4.fc40.noarch                7/23
  Installing       : python3-pyyaml-6.0.1-14.fc40.x86_64                   8/23
  Installing       : python3-ply-3.11-23.fc40.noarch                       9/23
  Installing       : python3-pycparser-2.20-14.fc40.noarch                10/23
  Installing       : python3-cffi-1.16.0-4.fc40.x86_64                    11/23
  Installing       : python3-cryptography-41.0.7-1.fc40.x86_64            12/23
  Installing       : python3-packaging-23.2-4.fc40.noarch                 13/23
  Installing       : python3-markupsafe-2.1.3-4.fc40.x86_64               14/23
  Installing       : python3-jinja2-3.1.6-1.fc40.noarch                   15/23
  Installing       : ansible-core-2.16.14-1.fc40.noarch                   16/23
  Running scriptlet: libutempter-1.2.1-13.fc40.x86_64                     17/23
  Installing       : libutempter-1.2.1-13.fc40.x86_64                     17/23
  Installing       : util-linux-2.40.2-1.fc40.x86_64                      18/23
  Running scriptlet: util-linux-2.40.2-1.fc40.x86_64                      18/23
  Installing       : openssh-9.6p1-2.fc40.x86_64                          19/23
  Installing       : libcbor-0.11.0-1.fc40.x86_64                         20/23
  Installing       : libfido2-1.14.0-4.fc40.x86_64                        21/23
  Installing       : openssh-clients-9.6p1-2.fc40.x86_64                  22/23
  Running scriptlet: openssh-clients-9.6p1-2.fc40.x86_64                  22/23
  Installing       : ansible-9.13.0-1.fc40.noarch                         23/23
  Running scriptlet: ansible-9.13.0-1.fc40.noarch                         23/23

Installed:
  ansible-9.13.0-1.fc40.noarch
  ansible-core-2.16.14-1.fc40.noarch
  fmt-10.2.1-5.fc40.x86_64
  libcbor-0.11.0-1.fc40.x86_64
  libdnf5-5.1.17-4.fc40.x86_64
  libedit-3.1-54.20250104cvs.fc40.x86_64
  libfdisk-2.40.2-1.fc40.x86_64
  libfido2-1.14.0-4.fc40.x86_64
  libutempter-1.2.1-13.fc40.x86_64
  openssh-9.6p1-2.fc40.x86_64
  openssh-clients-9.6p1-2.fc40.x86_64
  python3-argcomplete-3.6.2-2.fc40.noarch
  python3-cffi-1.16.0-4.fc40.x86_64
  python3-cryptography-41.0.7-1.fc40.x86_64
  python3-jinja2-3.1.6-1.fc40.noarch
  python3-libdnf5-5.1.17-4.fc40.x86_64
  python3-markupsafe-2.1.3-4.fc40.x86_64
  python3-packaging-23.2-4.fc40.noarch
  python3-ply-3.11-23.fc40.noarch
  python3-pycparser-2.20-14.fc40.noarch
  python3-pyyaml-6.0.1-14.fc40.x86_64
  python3-resolvelib-1.0.1-4.fc40.noarch
  util-linux-2.40.2-1.fc40.x86_64

Complete!
17 files removed
--> 10be93c5affb
STEP 5/5: ADD ansible.cfg /etc/ansible/
COMMIT

--> 4e2504d7fe11
4e2504d7fe114cf99152975db619f22087db448947c135fc6e4a4e3b6e02768d
```

rpoisel commented:
> @rpoisel could you update the fedora image version. After that i think we are good

Hi Ryan, thanks for your support! I just changed the docker base image from Fedora 36 to 42. Please let me know if you need any other changes in this PR.

cooktheryan commented:
I think we will need to update crun

```
- name: Build compatible crun
  run: |
    # Install build dependencies
    sudo apt-get update
    sudo apt-get install -y autoconf automake gcc git-core glibc-static \
      libcap-dev libseccomp-dev libtool make pkg-config python3 \
      libsystemd-dev libyajl-dev
    
    # Clone and build crun
    git clone https://github.com/containers/crun.git
    cd crun
    git checkout v1.14.4  # Use stable version
    ./autogen.sh
    ./configure --prefix=/usr/local
    make -j$(nproc)
    sudo make install
    
    # Verify installation
    /usr/local/bin/crun --version

- name: Build Podman with compatible runtime
  run: |
    export PATH="/usr/local/bin:$PATH"
    # Your existing Podman build commands here
    make
    sudo make install
    ```
    
    

To be compatible with the podman version
